site/planning.tmpl

```diff
@@ -0,0 +1,114 @@
+{{ end }}
+
+{{ define "style" }}
+  <link rel="stylesheet" type="text/css" href="//cdn.datatables.net/1.10.19/css/jquery.dataTables.css">
+  <link rel="stylesheet" href="/third_party/datatables-bulma/dataTables.bulma.css" />
+  <link rel="stylesheet" href="/static/css/kanban.css?{{.Version}}">
+{{ end }}
+
+{{define "subnav"}}
+
+<nav class="navbar secondary" role="navigation" aria-label="secondary navigation">
+  <div class="navbar-secondary-brand">
+  </div>
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-center">
+          <div class="right-item">
+          <div class="tab-link"><a href="#" title="open in new tabs" onclick="openAllTabs(); return false;"><i class="fas fa-external-link-alt"></i></a></div>
+          <span title="Data as of {{ .ResultAge | HumanDuration}} ago">{{ if eq .TotalShown .Total }}{{ .Total }} unique items{{ else }}Showing {{ .TotalShown }} of {{ .Total}} unique items{{ end }},
+          Avg age: {{ .CollectionResult.AvgAge | toDays }}
+          {{ if .VelocityStats }}, Historical closure rate: <a href="/s/{{.VelocityStats.Collection.ID }}">{{ printf "%.1f" $.ClosedPerDay }} issue(s) per day</a>{{ end }}
+          </span>
+          <span class="alt-view"><a href="/s/{{ .ID }}{{ $.GetVars }}">Items</a></span>
+          </div>
+          <script>
+          function openAllTabs() {
+              {{ range .UniqueItems}}window.open("{{ .URL | toJS }}", "_tab{{ .ID }}");
+              {{ end }}
+          }
+          </script>
+
+    </div>
+  </div>
+    <div class="navbar-right">
+      <div class="navbar-form">
+          {{ if .SelectorOptions }}
+            <div class="buttons">
+                Milestone:
+                <form style="display: inline-block;" action="/k/{{ .ID }}" method="get">
+                    <select onchange="this.form.submit();" name="{{ .SelectorVar }}">
+                      {{ range .SelectorOptions }}
+                        <option value="{{ .Value }}" {{ if .Selected }}selected{{ end }}>{{ .Text }}</option>
+                      {{ end }}
+                    </select>
+                </form>
+            </div>
+          {{ end }}
+      </div>
+    </div>
+</nav>
+{{ end }}
+
+{{define "content"}}
+  {{ if .CollectionResult.RuleResults }}
+
+    {{ if ne .Description "" }}
+      <div class="box description">
+      <pre>{{ .Description }}</pre>
+      </div>
+    {{ end }}
+
+    <div class="box outcome kanban">
+      <table id="kanban-table" class="compact is-size-6">
+      <thead>
+        <tr>
+          <th class="hd" id="status-col">Status</th>
+          {{- range .CollectionResult.RuleResults }}
+          <th class="hd" id="{{ .Rule.ID | Class  }}" title="{{ .Rule | toYAML }}">{{ .Rule.Name}}</th>
+          {{ end }}
+        </tr>
+      </thead>
+      <tbody>
+
+      {{ $col := .Collection }}
+
+      {{ range .Swimlanes }}
+      <tr class="swimlane {{ if eq .Name "Not Planned" }}unassigned-lane{{ end }}">
```

Comment:
Hardcoding this string feels brittle. Is there another proxy we could use to determine it?

Alternatively, you could add a helper function like `isUnassigned` that will allow you to compare the name against the `unplanned` constant.

Reply:
Was being lazy. Will fixit.